swagger-confluence-client.yaml

```diff
@@ -44,6 +44,33 @@ paths:
           description: "Returned if there is no space with the given key, or if the calling user does not have permission to view the space."
           schema:
             $ref: '#/definitions/confluenceSpaceError'
+  /content/{pageId}:
+    get:
+      summary: "Gets a confluence page content by id."
+      operationId: "getPage"
+      tags:
+        - confluence
+      description: "Returns a piece of Content."
+      security:
+        - crowdAuth: []
+      parameters:
+        - name: 'pageId'
+          type: "integer"
+          in: "path"
+          description: "The page ID to return information about the page."
+          required: true
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "Returns a full JSON representation of a piece of content"
+          schema:
+            $ref: '#/definitions/confluencePageExists'
```

Comment:
...Exists implies it's a check and will return a boolean, this will return a full page.

Reply:
Aye. I rename it into confluencePageRepresentation.